romanesco/__main__.py

```diff
@@ -23,6 +23,7 @@ def run(*pargs, **kwargs):
                         url=jobInfo.get('url'), method=jobInfo.get('method'),
                         headers=jobInfo.get('headers')) as jm:
             kwargs['_job_manager'] = jm
```

Comment:
We do that since it is passed as a magic variable into the script env in python mode. It doesn't necessarily need to be underscore prefixed here, though.